Tests/varLib/varLib_test.py

```diff
@@ -558,6 +559,9 @@ def test_varlib_build_sparse_masters_MVAR(self):
             varfont, _, _ = build(ds)
             mvar_tags = [vr.ValueTag for vr in varfont["MVAR"].table.ValueRecord]
             assert all(tag in mvar_tags for tag in fontTools.varLib.mvar.MVAR_ENTRIES)
+        finally:
+            for master in master_fonts:
+                master.close()
```

Comment:
aren't these being closed by default inside varLib.build? why you need to close them again

Reply:
Ahh, you're right. I saw `load_masters` and didn't look far enough to see that `build` was actually called.